client/src/components/FormInput/FormInputRight.tsx

```diff
@@ -0,0 +1,67 @@
+import { FormControl, InputBase, InputLabel, MenuItem, Select } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+const StyledInput = styled(InputBase)(({ theme }) => ({
```

Comment:
The exact same styled component has been defined in another file, we can reuse the existing one?